ontologies/works.ttl

```diff
@@ -0,0 +1,344 @@
+	rdfs:domain ww:Work ;
+	rdfs:range xsd:dateTime ;
+	rdf:isDefinedBy <TBC/works> .	
+
+ww:identifier rdf:type owl:DatatypeProperty ;
+	rdfs:label "identifier"@en ;
+	rdfs:comment "Relates a work to an identifier such as a manuscript number"@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .
+
+ww:title rdf:type owl:DatatypeProperty ;
+	rdfs:label "title"@en ;
+	rdfs:comment "The title or other short label of a work, including labels not present in the actual work or item but applied by the cataloguer for the purposes of search or description."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .	
+	
+ww:identifier rdf:type owl:DatatypeProperty ;
+	rdfs:label "identifier"@en ;
+	rdfs:comment "Relates a work to an identifier such as a manuscript number"@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:level rdf:type owl:DatatypeProperty ;
+	rdfs:label "level"@en ;
+	rdfs:comment "Indicates the position of a work within a hierarchy of works: for example, Collection, Series, Item etc. Item in CALM descriptions is taken to mean an individual work, which is then manifested in a physical or digital item."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:placeOfPublication rdf:type owl:DatatypeProperty ;
+	rdfs:label "placeOfPublication"@en ;
+	rdfs:comment "Relates a work to the place at which it was formally published, in the case of published material."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:placeOfCreation rdf:type owl:DatatypeProperty ;
+	rdfs:label "placeOfCreation"@en ;
+	rdfs:comment "Relates a work to the place at which it was created, in the case of unpublished material."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .
+
+ww:description rdf:type owl:DatatypeProperty ;
+	rdfs:label "description"@en ;
+	rdfs:comment "General descriptive information about the work and its content."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .	
+	
+ww:alternativeIdentifier rdf:type owl:DatatypeProperty ;
+	rdfs:label "alternativeIdentifier"@en ;
+	rdfs:comment "Relates the work to an identifier that is not preferred, such as a previous reference."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .	
+	
+ww:systemIdentifier rdf:type owl:DatatypeProperty ;
+	rdfs:label "systemIdentifier"@en ;
+	rdfs:comment "Relates the work to an identifier that is not made visible to the public but used to generate hierarchy."@en ;
+	rdfs:domain ww:Work ;
+	rdfs:range rdf:langString ; 
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:systemNumber rdf:type owl:DatatypeProperty ;
+	rdfs:label "systemNumber"@en ;
+	rdfs:comment "Relates the work's metadata to an system-generated GUID."@en ;
```

Comment:
I'm not sure the intent here, but just the naming makes me uncomfortable. The identifier discussed in #31 will be in the URI. We may also have a separate property for us to use internally for ease of access, but the reality is that we will probably use the aliased "id" property for that in the JSON.

In other words, internally we'll be treating the JSON the same as other non-RDF clients, blissfully unaware that if we parse the @context and add the @base, that id is actually @id and is a URI. If that makes any sense!

I'd drop this predicate for now, or rename if @ChristopherHilton can tell us what it is...